upgrades/3.x/3.5.14/1-fix-cors-env-vars.js

```diff
@@ -0,0 +1,33 @@
+const duplicatedKeysForConsole = [
+    'portal.http.cors.allow-origin',
+    'portal.http.cors.allow-headers',
+    'portal.http.cors.allow-methods',
+    'portal.http.cors.exposed-headers',
+    'portal.http.cors.max-age',
+    'console.http.cors.allow-origin',
+    'console.http.cors.allow-headers',
+    'console.http.cors.allow-methods',
+    'console.http.cors.exposed-headers',
+    'console.http.cors.max-age'
+];
+
+print('Rename portal/console CORS parameters');
+db.parameters.find({ referenceId: { $exists: true }}).forEach(parameter => {
+    if (duplicatedKeysForConsole.includes(parameter._id.key)) {
+        const parameterKey = parameter._id.replace('portal.http.cors', 'http.api.portal.cors')
+          .replace('console.http.cors', 'http.api.management.cors');
+        const parameterUpdated = {
+            _id: {
+                key: parameterKey,
+                referenceId: 'DEFAULT',
+                referenceType: 'ORGANIZATION',
```

Comment:
what if the reference id is not 'DEFAULT'
maybe we should keep `parameter._id.referenceId`

wdyt ?